inference-engine/include/vpu/hddl_plugin_config.hpp

```diff
@@ -3,10 +3,11 @@
 //
 
 /**
+ * @deprecated Use vpu/hddl_config.hpp instead.
  * @brief A header that defines advanced related properties for VPU plugins.
  * These properties should be used in SetConfig() and LoadNetwork() methods of plugins
  *
- * @file 
+ * @file hddl_plugin_config.hpp
  */
```

Comment:
Should there be `#pragma warning`?

Reply:
For GCC compiler we can add `#pragma GCC warning "message"`, but it will break the compilation while `-Werror` is set.